vm/src/obj/pystr.rs

```diff
@@ -297,4 +297,69 @@ pub trait PyCommonString<E> {
             &self
         }
     }
+
+    fn py_splitlines<U8, FW, W>(
+        &self,
+        bytes: impl std::iter::Iterator<Item = U8>,
```

Comment:
Unfortunately, `str::bytes()` is `std::iter::Itarator<Item=u8>` but `[u8]::iter()` is `std::iter::Iterator<Item=&u8>`. Can I do this without `CopyTo` trait?

Reply:
You could probably just give it a byte slice using `str.as_bytes()`

Reply:
Thanks, it works. I am still curious how to write a generic function to take `T` and `&T` when T is a Copy. it is practically same but I couldn't find an easy way with std.